assets/html/scss/documenter/components/_docstring.scss

```diff
@@ -48,6 +48,10 @@
       @extend .is-primary;
       @extend .is-size-7;
     }
+
+    .math-container {
+      overflow-x: auto;
+    }
```

Comment:
I don't think this should be in `_docstring.scss`, since we want this to apply in all circumstances (currently the example cases don't get scrollbars)? I think the right place would be in the `.docs-main > article` selector: https://github.com/JuliaDocs/Documenter.jl/blob/60f368c0feb0cd17dafec13af2c83fbbaf948499/assets/html/scss/documenter/layout/_main.scss#L5-L10